common-rendering/src/components/keyEvents.tsx

```diff
@@ -201,10 +203,19 @@ const timeStyles = (supportsDarkMode: boolean): SerializedStyles => css`
 `;
 
 const ListItem = ({ keyEvent, theme, supportsDarkMode }: ListItemProps) => {
+	useTimeAgo();
```

Comment:
Should we only be calling `useTimeAgo()` once per layout? if so is it better to lift it to `KeyEvents` for now, and add a TODO that it should be lifted to the overall layout at some point in the future?

Reply:
This is an interesting point - do we need to consider at what level we're going to hydrate the component/s too? In AR previously we've tried to keep hydration as minimal as possible, esp as Emotion styling doesn't work client-side.

Reply:
Olly's right, `useTimeAgo` sets up a `setInterval` on the whole page so needs to be called quite high up, probably in `App.tsx`

Reply:
> do we need to consider at what level we're going to hydrate the component/s too?

Yeah, completely this. Hydration is a _big_ deal and really needs to be thought through. This is a little off topic (for this PR) but I was chatting with @lucymonie about this. I don't really have strong opinions on solutions but we do need to keep the risk of the TTI value increasing on the page if we hydrate the whole article body (which might be needed if we use React state to keep the blocks list live)

Reply:
Maybe this is one to discuss in planning later? For now I think lifting it to `KeyEvents` and adding a `TODO` makes sense.

Reply:
Even with a TODO statement we're setting ourselves up for problems later - `useTimeAgo` is very much a page level execute once action. If it is run twice on the page we might see strange, unintended behaviour.

It's true though that one benefit to adding it here now is that the values are 'live' in storybook. This helps testing. But if that were a desired outcome then maybe we could call it from inside `preview.js`?

Reply:
That's an excellent idea, given it's the `preview.js` for the entire repo is there any risk of it effecting any of the DCR components?

Reply:
Ha, `hooks can only be called in the body of a function component`

Reply:
Added to the story.

Reply:
I _think_ you can run the hook globally using a [global decorator](https://storybook.js.org/docs/react/writing-stories/decorators#global-decorators). That should set up the function for all stories, so any time anyone has a `time` element that respects the contract, they will get updated!

Reply:
Updated to use a global decorator 👍